src/GraphQL/Transports/Json/WebSocketMessage.cs

```diff
@@ -0,0 +1,24 @@
+namespace GraphQL.Transports.Json
+{
+    public class WebSocketMessage
+    {
+        public const string ID_KEY = "id";
+        public const string TYPE_KEY = "type";
+        public const string PAYLOAD_KEY = "payload";
+
+        /// <summary>
+        /// Nullable Id
+        /// </summary>
+        public string? Id { get; set; }
```

Comment:
You removed all `DataMember`/`DataContract` attributes. I remember that those attributes are required for proper serialization by Newtonsoft.Json. Did you do it intentionally? Everything works without them?

Reply:
The `DataMember` and `DataContract` attributes are `Newtonsoft.Json` specific.  There are other `System.Text.Json`-specific attributes.  In order to be able to remove the attributes, a custom `JsonConverter` was added to each serialization project for each class.  I see no other reasonable way to abstract the transport from the data classes.

Reply:
May be related to https://github.com/graphql-dotnet/server/pull/321

Reply:
And https://github.com/graphql-dotnet/server/pull/194  but I think that actually these attributes can be safely removed.

Reply:
Yes, very similar to those issues.  I'm sure they can.  I've already written tests that verify this.

Reply:
I thought so.